.github/workflows/macos-build.yaml

```
@@ -4,15 +4,18 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - macos-11-runner
```

Don't forget to change this back to master.
```suggestion
      - master
```

---

@nwt: Thanks for the reminder!